faq.html

```diff
@@ -300,36 +300,24 @@ <h2>Supported Features</h2>
 				</div>
 				
 				<div class="faq_item">
 					<div class="faq_question">Does Tribler have a WebUI?</div >
-					<div class="faq_answer"><p>There is currently an experimental feature in Tribler that implements the protocol used by the open source webUI of  uTorrent. You can enable the webUI under Settings, Experimental.</p></div>
-				</div>
-				
-				<h2>Reputation</h2>
-				<div class="faq_item">
-					<div class="faq_question">I've got a bad reputation!</div >
-					<div class="faq_answer"><p>Don't panic! A good reputation will only get you a little boost from other Tribler peers, not having a good reputation will be the same as downloading using any other BitTorrent client.</p>
-					<p>If you wish to improve your reputation, you can leave your client on to seed after you've finished downloading. If you want to improve your reputation even further, you can try and find popular content and seed that (after downloading it yourself, of course).</p></div>
-				</div>
-				
-				<div class="faq_item">
-					<div class="faq_question">Can I copy or share my reputation?</div >
-					<div class="faq_answer"><p>You can copy your reputation by copying the complete settings directory of Tribler. You can't share your reputation, since it's based on a unique identifier generated for you client.</p></div>
+					<div class="faq_answer"><p>There is currently no way to run Tribler in a browser. However, you can use our RESTful API, built for developers, to communicate with the Tribler core. See <a href="http://tribler.readthedocs.io/en/latest/restapi/introduction.html">here</a> for the documentation of this API.</p></div>
 				</div>
-				
+								
 				<h2>Miscellaneous</h2>
 				<div class="faq_item">
 					<div class="faq_question">What's with the collected torrent files?</div >
 					<div class="faq_answer"><p>The collected torrent files are collected while you are online. They are part of the distributed library of torrent files you search through when you search using Tribler. All Tribler peers in the network participate in that large library by exchanging torrents and keeping a part of that library available by storing it locally.</p>
-					<p>The folder with collected torrent files will at first be filled to contain up to 50?000 .torrent files (50 000 .torrents files are actually stored in 150 000 files, as each file is accopanied by a hash, and tree). After that Tribler will now and then throw away a part of that collection and find some others to store locally, keeping the library up to date.</p></div>
+					<p>The folder with collected torrent files will at first be filled to contain up to 50?000 .torrent files (50 000 .torrents files are actually stored in 150 000 files, as each file is accompanied by a hash, and tree). After that Tribler will now and then throw away a part of that collection and find some others to store locally, keeping the library up to date.</p></div>

```

Comment:
Typo: 50?000

Reply:
Nice find: I think we should have all numbers comma separated.

Reply:
Thanks, I've changed the formatting of these numbers.